packages/doctype-tile/src/tile-doctype.ts

```diff
@@ -31,6 +32,21 @@ export interface TileMetadataArgs {
   deterministic?: boolean
 }
 
+interface CeramicCommon {
+    logger?: DiagnosticsLogger
+}
+
+/**
+ * Interface for an object that contains a DID that can be used to sign Ceramic commits.
+ * Any implementation of CeramicAPI will match this interface, though if no CeramicAPI instance is
+ * available users can provide any object containing an authenticated DID instance.
+ */
+interface CeramicSigner extends CeramicCommon {
```

Comment:
I left this unexported and local to tile-doctype.ts for now.  I figure we can move it if/when we need it somewhere else.  If you'd like me to move it somewhere more reusable now though just lmk

Reply:
I changed my mind on this and moved it into `common` so that `CeramicApi` can extend `CeramicSigner` to make it super clear that a `CeramicApi` instance can be used anywhere that asks for a `CeramicSigner`